CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ArrayRequesthandler.swift

```diff
@@ -22,7 +22,7 @@ public class ArrayRequestHandler {
         case "array_create":
             let array: [Any]? = args.get(name: "content_array")!
             if array != nil {
-                return MutableArrayObject(data: array)
+                return MutableArrayObject(data: array!)
             } else {
                 return MutableArrayObject()
```

Comment:
is this for adding coverage to `MutableArrayObject()` and `MutableArrayObject(data:)` constructors? if not, you can skip the nullability and use single `MutableArrayObject(data: array)`, in case of empty array or array with values.